Sources/CountLabel.swift

```diff
@@ -1,9 +1,81 @@
-//
-//  CountLabel.swift
-//  Bakken & Baeck
-//
-//  Created by Marijn Schilling on {TODAY}.
-//  Copyright © 2017 Bakken & Baeck. All rights reserved.
-//
-
-import Foundation
+import UIKit
+
+open class CountLabel: UILabel {
+    open var postfix: String?
+    open var prefix: String?
+
+    var currentValue: Int {
+        set {
+        }
+        get {
+            if self.progress >= self.totalTime {
+
+                return self.endValue
+            }
+
+            let percent = self.progress / self.totalTime
+
+            return Int(Double(self.startValue) + (percent * Double(self.endValue - self.startValue)))
```

Comment:
I'm 95% sure this is optimised by the compiler. We can do an instruments run and see. 👍